src/item.cpp

```diff
@@ -3531,6 +3531,15 @@ std::set<fault_id> item::faults_potential() const
     return res;
 }
 
+int item::wheel_area() const
+{
+    if( !is_wheel() ) {
```

Comment:
overly verbose. `item.cpp` is already obese.

`return is_wheel() ? type->wheel->diameter * type->wheel->width : 0`